in_toto/models/layout.py

```diff
@@ -256,13 +266,10 @@ def _validate_type(self):
 
   def _validate_threshold(self):
     """Private method to check that the threshold field is set to an int."""
-    try:
-      # int(self.threshold)
-      # FIXME: we don't have support for threshold yet
-      pass
-    except Exception as e:
+    if type(self.threshold) != int:
       raise securesystemslib.exceptions.FormatError(
-          "Invalid threshold value for this step. Exception: {}".format(e))
+          "Invalid threshold '{}' value, must be an int."
+          .format(self.threshold))
```

Comment:
I have already changed this before. My fork reflects this change too. It's strange that you encountered this.